.ci/README.md

```diff
@@ -32,80 +32,128 @@ It's possible that a consumer of the e2e tests would need to define a specific l
+
+> The IP address of the stack in the `.stack-host-ip` file will be used by the automation.
+
+Please remember to [destroy the stack](#destroying-the-stack-and-the-test-node) once you finished your testing.
+
+### Create and configure test node
+
+There are different VM flavours that you can use to run the Elastic Agent and enrol it into the Stack: Debian, CentOS, SLES15, Oracle Linux... using AMD and ARM as architecture. You can find the full reference of the platform support [here](https://github.com/elastic/e2e-testing/blob/4517dfa134844f720139d6bab3955cc8d9c6685c/.ci/.e2e-platforms.yaml#L2-L42).
+
+In these VMs, the test framework will download a binary to install the Elastic Agent (TAR files, DEB/RPM packages...), and will execute the different agent commands to install, enroll, uninstall, etc.
+
+It's possible to configure the test node (OS, architecture), using the values that are already present in [the platforms descriptor](.e2e-platforms.yaml):
 
-### Setup stack
+```shell
+# example for Centos 8 ARM 64
+export NODE_IMAGE="ami-01cdc9e8306344fe0"
+export NODE_INSTANCE_TYPE="a1.larg"
+export NODE_LABEL="centos8_arm64"
+export NODE_USER="centos"
+```
+
+Besides that, it's possible to configure the test node for the different test suites that are present in the test framework: `fleet`, `helm` and `kubernetes-autodiscover`. Please configure the test node setting the suite, being `fleet` the default:
 
+```shell
+# all possible suites
+export SUITE="fleet"
+export SUITE="helm"
+export SUITE="kubernetes-autodiscover"
 ```
-> venv/bin/ansible-playbook .ci/ansible/playbook.yml \
-    --private-key="$HOME/.ssh/id_rsa" \
-    --extra-vars "nodeLabel=stack nodeImage=ami-0d90bed76900e679a nodeInstanceType=c5.4xlarge" \
-    --extra-vars "runId=$RUN_ID workspace=$HOME/Projects/e2e-testing/ sshPublicKey=$HOME/.ssh/id_rsa.pub" \
-    --ssh-common-args='-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' \
-    -t setup-stack \
-    -i <ip address above>,
+
+Finally, please create the test node:
+
+```shell
+export SSH_KEY="PATH_TO_YOUR_SSH_KEY_WITH_ACCESS_TO_AWS"
+export SUITE="fleet"
+make -C .ci create-node
 ```
 
-**Note**: The comma at the end of the ip address is required.
+A `.node-host-ip` file will be created in the `.ci` directory of the project including the IP address of the node instance. Check it out from that file, or make a note of the IP address displayed in the ansible summary, as you'll probably need it to SSH into it for troubleshooting.
+
+> The IP address of the node in that file will be used by the automation.
 
-### Deploy test node
+Please remember to [destroy the node](#destroying-the-stack-and-the-test-node) once you finished your testing.
 
+### Run a test suite
+
+You can select the specific tags that you want to include in the test execution. Please look for the different tags in the existing feature files for the suite you are interested in running:
```

Comment:
[Suggestion]
Show or state where the tags are defined. From my experience, the first time I saw this tags being referenced, I had no idea where they were defined, therefore I did not know what to set them to

Reply:
+1 to this suggestion. I have had this issue in the past.

Reply:
Indeed. Adding it to the docs:

> You can select the specific tags that you want to include in the test execution. Please look for the different tags in the existing feature files for the suite you are interested in running. For that, please check out the tags/annotations that are present in those feature files (`*.feature`), which live in the `features` directory under your test suite. In example, for `fleet` test suite, you can find them [here](../e2e/_suites/fleet/features/).